frontend/src/scenes/instance/DeadLetterQueue/DeadLetterQueue.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { PageHeader } from 'lib/components/PageHeader'
+import { SceneExport } from 'scenes/sceneTypes'
+import { Tabs } from 'antd'
+import { useValues, useActions } from 'kea'
+import { deadLetterQueueLogic, DeadLetterQueueTab } from './deadLetterQueueLogic'
+import { userLogic } from 'scenes/userLogic'
+import { MetricsTab } from './MetricsTab'
+
+export const scene: SceneExport = {
+    component: DeadLetterQueue,
+    logic: deadLetterQueueLogic,
+}
+
+const { TabPane } = Tabs
+
+export function DeadLetterQueue(): JSX.Element {
+    const { user } = useValues(userLogic)
+    const { activeTab } = useValues(deadLetterQueueLogic)
+    const { setActiveTab } = useActions(deadLetterQueueLogic)
+
+    if (!user?.is_staff) {
+        return <></>
```

Comment:
Don't we have some default "You need to be staff to see this page" + link to docs on how to get to staff status?